### PR TITLE
Fix tests CMA

### PR DESCRIPTION
**Pull request by DrTaDa**

### CODE REVIEW

**Comment by DrTaDa** [comment]
@wvangeit to fix the issue with #393

File: tox.ini [hunk 1]
```diff
@@ -19,7 +19,7 @@ deps =
     coverage
     flake8
     mock
-    neuron-nightly
+    neuron
```
Comment (wvangeit) [inline tox.ini:22]: I'd prefer keep on testing on nightly though, to find issues as early as possible

Comment (DrTaDa) [reply tox.ini:22]: I think we just found an issue then :D

Comment (DrTaDa) [reply tox.ini:22]: https://github.com/BlueBrain/BluePyOpt/actions/runs/2549275161

Comment (DrTaDa) [reply tox.ini:22]: Is it something we have to report to the nrn github ?

Comment (wvangeit) [reply tox.ini:22]: Probably best. This is an error generator by a neuron header  file, right?

Comment (DrTaDa) [reply tox.ini:22]: The issue is coming from this line: https://github.com/BlueBrain/BluePyOpt/blob/cc5448730d73311150421a4bec0930315d39cf9b/examples/stochkv/mechanisms/StochKv3.mod#L110

Comment (DrTaDa) [reply tox.ini:22]: Ah Pramod answered: https://github.com/neuronsimulator/nrn/issues/1875

Comment (pramodk) [reply tox.ini:22]: As mentioned in the [BBP internal slack channel](https://bluebrainproject.slack.com/archives/C02UDSR3AAY/p1655995850804189), assuming this StochKv3.mod from BBP, I would copy the updated version from BBP models (e.g. [here](https://bbpgitlab.epfl.ch/hpc/sim/models/neocortex/-/blob/main/mod/v6/StochKv3.mod))

Would be happy to help if anything else is required.

**Comment by wvangeit** [comment]
I see that there are quite a few changes in examples/l5pc/mechanisms/ProbAMPANMDA_EMS.mod and GABA. I think we don't use these files? Maybe it's better to just remove these from the repo

**Comment by DrTaDa** [comment]
We use both in the examples/stochkv

**Comment by wvangeit** [comment]
Ok, but not the synapse files i assume, right?

File: bluepyopt/tests/test_deapext/test_optimisationsCMA.py [hunk 2]
```diff
@@ -57,4 +58,5 @@ def test_optimisationsCMA_MO_run():
 
     assert abs(log.select("avg")[-1] - 40.) < 1e-4
     assert abs(log.select("std")[-1] - 16.32993) < 1e-4
-    assert pop[0] == [0.09601241274168831, 0.024646650865379722]
+    assert abs(pop[0][0] - 0.09601241274168831) < 1e-6
```
Comment (mgeplf) [inline bluepyopt/tests/test_deapext/test_optimisationsCMA.py:61]: FYI: https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-approx

**Review by wvangeit — APPROVED** [review]

**Pull request merged**